salt/utils/cloud.py

```diff
@@ -88,6 +88,8 @@
 except ImportError:
     HAS_GETPASS = False
 
+MSGPACK_ENCODING = 'utf-8'
```

Comment:
Add a comment required to support international characters in tags. for example.